homeassistant/components/volvooncall.py

```diff
@@ -192,6 +213,7 @@ def assumed_state(self):
     @property
     def device_state_attributes(self):
         """Return device specific state attributes."""
-        return dict(model='{}/{}'.format(
-            self.vehicle.vehicle_type,
-            self.vehicle.model_year))
+        return dict(self.instrument.attributes,
```

Comment:
What attributes are we presenting here?

Reply:
Currently additional attributes are only provided for the sensor displaying the last trip: start address, end address, trip duration etc (with the main state being the timestamp of the trip).

Reply:
Absolute timestamps are ok, and they should be UTC. Relative timestamps should not be calculated from current time, since that would have them change at each update. If that's not the case it's fine.
https://developers.home-assistant.io/docs/en/creating_platform_code_review.html#5-entity

Reply:
✓